Formula/eventql.rb

```diff
@@ -0,0 +1,25 @@
+class Eventql < Formula
+  desc "Database for large-scale event analytics"
+  homepage "https://eventql.io"
+  url "http://dl.eventql.io/eventql/v0.3.2/eventql-0.3.2.tgz"
```

Comment:
Let's try to use TLS: https://github.com/eventql/eventql/releases/download/v0.3.2/eventql-0.3.2.tgz.